doc/sedcli.8

```diff
@@ -51,7 +51,7 @@ update Admin1 password for Locking SP.
 .SH OPTIONS
 
 .IP "\fB\-D -d <device> [-f {normal|udev}]\fR or \fB\-\-discovery --device <device> [--format {normal|udev}]\fR"
-Performs Level 0 Discovery and prints out info in human-readable (default) or
+Provides level0 discovery information and tper properties of the device. Prints the info in human-readable (default) or
```

Comment:
"Performs Level 0 and Level 1 Discovery and prints out info in human-readable...."